apps/fluent-tester/src/TestComponents/Input/InputDefault.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import { View } from 'react-native';
+
+import type { IconProps } from '@fluentui-react-native/icon';
+import { Input } from '@fluentui-react-native/input';
+
+import DismissSvg from '../../../assets/dismissIcon.svg';
+import FilledSvg from '../../../assets/filledIcon.svg';
+import OutlineSvg from '../../../assets/outlineIcon.svg';
+
+export const dismissIconProps: IconProps = { svgSource: { src: DismissSvg, viewBox: '0 0 20 20' } };
+export const outlineIconProps: IconProps = { svgSource: { src: OutlineSvg, viewBox: '0 0 20 20' } };
+export const filledIconProps: IconProps = { svgSource: { src: FilledSvg, viewBox: '0 0 20 20' } };
+
+export const InputDefault: React.FunctionComponent = () => {
+  return (
+    <View>
+      <Input value="Value prop passed!" />
```

Comment:
Consider expanding the tester code to demonstrate usage of value component and error handling props - yo can choose it as a separate PR

Reply:
Updated tests